docs/integrations-guides/rollup-guides/op-stack/README.md

```diff
@@ -34,32 +37,38 @@ In the op-node `rollup.json` configuration the following should be set:
 
 ```json
 {
-  "plasma_config": {
-    "da_challenge_contract_address": "0x0000000000000000000000000000000000000000",
+  "alt_da": {
     "da_commitment_type": "GenericCommitment",
+    "da_challenge_contract_address": "0x0000000000000000000000000000000000000000",
     "da_challenge_window": 300,
     "da_resolve_window": 300
   }
 }
 ```
+Only `da_commitment_type` is important, because eigenDA does not use da challenges.
+
 
 #### op-node CLI configuration
 
 The following env config values should be set to ensure proper communication between op-node and eigenda-proxy, replacing `{EIGENDA_PROXY_URL}` with the URL of your EigenDA Proxy server.
 
-- `OP_NODE_PLASMA_ENABLED=true`
-- `OP_NODE_PLASMA_DA_SERVICE=true`
-- `OP_NODE_PLASMA_VERIFY_ON_READ=false`
-- `OP_NODE_PLASMA_DA_SERVER={EIGENDA_PROXY_URL}`
+- `OP_NODE_ALTDA_ENABLED=true`
+- `OP_NODE_ALTDA_DA_SERVICE=true`
+- `OP_NODE_ALTDA_VERIFY_ON_READ=false`
+- `OP_NODE_ALTDA_DA_SERVER={EIGENDA_PROXY_URL}`
 
 #### op-batcher CLI configuration
 
 The following env config values should be set accordingly to ensure proper communication between OP Batcher and EigenDA Proxy, replacing `{EIGENDA_PROXY_URL}` with the URL of your EigenDA Proxy server.
 
-- `OP_BATCHER_PLASMA_ENABLED=true`
-- `OP_BATCHER_PLASMA_DA_SERVICE=true`
-- `OP_BATCHER_PLASMA_VERIFY_ON_READ=false`
-- `OP_BATCHER_PLASMA_DA_SERVER={EIGENDA_PROXY_URL}`
+- `OP_BATCHER_ALTDA_ENABLED=true`
+- `OP_BATCHER_ALTDA_DA_SERVICE=true`
+- `OP_BATCHER_ALTDA_VERIFY_ON_READ=false`
+- `OP_BATCHER_ALTDA_DA_SERVER={EIGENDA_PROXY_URL}`
+- `OP_BATCHER_ALTDA_MAX_CONCURRENT_DA_REQUESTS=1500`
```

Comment:
Should we use the multi-frames branch, so this number isn't that crazy

Reply:
Reworked on the paragraph... the new number is even more scary: [93dbba2](https://github.com/Layr-Labs/eigenda-docs/pull/20/commits/93dbba2d9834e0d807454e3c1b0d79cf87f0aa5f)